css-table/Table.html

```diff
@@ -0,0 +1,173 @@
+                        <td>name5@gmail.com</td>
+                        <td>
+                            <input type="radio" name="gender" value="male"> Male<br>
+                            <input type="radio" name="gender" value="female"> Female</td>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>name6</td>
+                        <td>name6@gmail.com</td>
+                        <td>
+                            <input type="radio" name="gender" value="male"> Male<br>
+                            <input type="radio" name="gender" value="female"> Female</td>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>name7</td>
+                        <td>name7@gmail.com</td>
+                        <td>
+                            <input type="radio" name="gender" value="male"> Male<br>
+                            <input type="radio" name="gender" value="female"> Female</td>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>name8</td>
+                        <td>name8@gmail.com</td>
+                        <td>
+                            <input type="radio" name="gender" value="male"> Male<br>
+                            <input type="radio" name="gender" value="female"> Female</td>
+                        </td>
+                    </tr>
+                    <tr>
+                        <td>name9</td>
+                        <td>name9@gmail.com</td>
+                        <td>
+                            <input type="radio" name="gender" value="male" checked> Male<br>
+                            <input type="radio" name="gender" value="female"> Female</td>
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+        </div>
+    </section>
+    <h3> Table with Rows of 2 different height and nohover :</h3>
+    <section>
+        <div class="container scroll-table">
+            <table>
+                <thead>
+                    <tr class="header">
+                        <th>
+                            <div>Name</div>
+                        </th>
+                        <th>
+                            <div>Email-ID</div>
+                        </th>
+                        <th>
+                            <div>Phone</div>
+                        </th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td>name1</td>
+                        <td>name1@gmail.com</td>
+                        <td>123456</td>
+                    </tr>
+                    <tr>
+                        <td>name2</td>
+                        <td>name2@gmail.com</td>
+                        <td>123456</td>
+                    </tr>
+                    <tr>
+                        <td height="50">name3</td>
```

Comment:
Remove the inline-height from all places

Reply:
Done